.goreleaser.yml

```diff
@@ -65,4 +46,4 @@ archive:
   # - Tag
   # - Version (Git tag without `v` prefix)
   # Default is ``
-  #name_template: "{{.ProjectName}}-v{{.Version}}"
+  #name_template: "{{.ProjectName}}-v{{.Version}}"
```

Comment:
Tiny nitpick: you should configure your editor to add a newline at the end of the last line of a file, or at the very least to not remove an existing one. You'll then avoid those meaningless whitespace/newline type of diff hunks, and more importantly, you'll be complying with the POSIX definition of a line!